experimental/Schemes/src/IdealSheaves.jl

```diff
@@ -216,6 +222,11 @@ function IdealSheaf(Y::AbsCoveredScheme,
 end
 
 
+function one_ideal_sheaf(X::AbsCoveredScheme)
+  dd = IdDict{AbsAffineScheme, Ideal}(U=>ideal(OO(U), elem_type(OO(U))[one(OO(U))]) for U in affine_charts(X))
+  return IdealSheaf(X, dd, check=false)
+end
+
```

Comment:
This already exists as `unit_ideal_sheaf`. We should not duplicate it.

Reply:
I was not aware. A search indeed supports the term `unit ideal` over `one ideal` .